iot-hub/Samples/service/DigitalTwinClientSamples/Thermostat/Program.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using CommandLine;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Devices.Samples
+{
+    public class Program
+    {
+        private static ILogger s_logger;
+
+        /// <summary>
+        /// A sample to illustrate how to perform root level service-side operations on a plug and play compatible device.
+        /// </summary>
+        /// <param name="args">
+        /// Run with `--help` to see a list of required and optional parameters.
+        /// </param>
+        /// <remarks>
+        /// This sample performs operations on a plug and play compatible device on the root level. For a sample on how to perform
+        /// operations on a component level on a plug an dplay compatible device, please check out <see href="https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/master/iot-hub/Samples/device">PnpDeviceSample</see>.
+        /// </remarks> 
+        public static async Task Main(string[] args)
+        {
+            // Parse application parameters
+            Parameters parameters = null;
+            ParserResult<Parameters> result = Parser.Default.ParseArguments<Parameters>(args)
+                .WithParsed(parsedParams =>
+                {
+                    parameters = parsedParams;
+                })
+                .WithNotParsed(errors =>
+                {
+                    Environment.Exit(1);
+                });
+
+            s_logger = InitializeConsoleDebugLogger();
+
+            s_logger.LogDebug("Set up the digital twin client.");
+            using DigitalTwinClient digitalTwinClient = DigitalTwinClient.CreateFromConnectionString(parameters.HubConnectionString);
+
+            s_logger.LogDebug("Set up and start the Thermostat sample.");
```

Comment:
nit: maybe call it out that we are starting the thermostat service sample? so as to not confuse it with the device sample